plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/editors/data/preferences/PrefPageResultSetMain.java

```diff
@@ -123,127 +127,149 @@ protected boolean supportsDataSourceSpecificOptions()
     protected Control createPreferenceContent(@NotNull Composite parent) {
         Composite composite = UIUtils.createPlaceholder(parent, 2, 5);
         Composite leftPane = UIUtils.createComposite(composite, 1);
-        leftPane.setLayoutData(new GridData(GridData.VERTICAL_ALIGN_BEGINNING));
+        leftPane.setLayoutData(new GridData(GridData.FILL_BOTH));
         Composite rightPane = UIUtils.createComposite(composite, 1);
-        rightPane.setLayoutData(new GridData(GridData.VERTICAL_ALIGN_BEGINNING));
-
-        {
-            Group queriesGroup = UIUtils.createControlGroup(leftPane, ResultSetMessages.pref_page_database_general_group_queries, 2, SWT.NONE, 0);
-            queriesGroup.setLayoutData(new GridData(GridData.VERTICAL_ALIGN_BEGINNING));
-
-            resultSetSize = UIUtils.createLabelText(queriesGroup, ResultSetMessages.pref_page_database_general_label_result_set_max_size, "0", SWT.BORDER);
-            resultSetSize.addVerifyListener(UIUtils.getIntegerVerifyListener(Locale.getDefault()));
-            resultSetSize.addFocusListener(new FocusAdapter() {
-                @Override
-                public void focusLost(FocusEvent e) {
-                    int newValue = CommonUtils.toInt(resultSetSize.getText());
-                    if (newValue > 0 && newValue < ResultSetPreferences.MIN_SEGMENT_SIZE) {
-                        resultSetSize.setText(String.valueOf(ResultSetPreferences.MIN_SEGMENT_SIZE));
-                    }
+        rightPane.setLayoutData(new GridData(GridData.FILL_BOTH));
+
+        // Group of queries settings
+        Group queriesGroup = UIUtils.createControlGroup(leftPane,
```

Comment:
Please bring back {}. They denote a scope. You don't need the `queriesGroup` variable outside. Same for blocks down below.